src/server/services/readMamacData.js

```diff
@@ -10,15 +10,15 @@ const Reading = require('../models/Reading');
 
 const parseCsv = promisify(csv.parse);
 
-function parseTimestamp(raw) {
+function parseTimestamp(raw, line) {
 	raw = raw.trim();
 	const timestampRegExp = /^\d{2}:\d{2}:\d{2} \d{1,2}\/\d{1,2}\/\d{1,2}$/;
 	if (!timestampRegExp.test(raw)) {
-		throw new Error(`Raw timestamp ${raw} does not pass regex validation`);
+		throw new Error('CSV line ' + line + `: Raw timestamp ${raw} does not pass regex validation`);
```

Comment:
``throw new Error(`CSV line ${line}: Raw timestamp ${raw} does not pass regex validation`);``
This will be more consistent since you don't have to concat the string.

Reply:
Makes sense! Fixed.